docs/source/ops/connect/helm.rst

```diff
@@ -0,0 +1,281 @@
+  Service for you, you need to set this. See reference section at the end of
+  this page for details.
+- ``authUrl``: If you want the Helm chart to provide either a Daml Driver for
+  PostgreSQL or a OAuth2 Middleware instance, you will need to set this to the
+  JWKS URL of your token provider.
+
+If you start the Trigger Service, you will need to configure it, as well as the
+OAuth2 Middleware. See the required options for them in the reference section
+at the end of this page.
+
+Finally, we also recommend looking at the ``resources`` option for each
+component and adjusting them to fit your particular use-case.
+
+Log Aggregation
+---------------
+
+All processes write their logs directly to stdout. This means that log
+aggregation can be addressed at the Kubernetes level and does not require any
+specific support from the Helm chart itself. One fairly easy way to achieve
+this is using `Filebeat <https://www.elastic.co/beats/filebeat>`_, which
+regulary collects the logs of your containers and ingests them into
+`Elasticsearch <https://www.elastic.co/elasticsearch/>`,
+`Logstash <https://www.elastic.co/logstash/>`_,
+`Kafka <https://kafka.apache.org/>`_, etc.
+
+You can find external documentation on, how to setup `ElasticSearch` with
+`Filebeat` and `Kibana` for analyzing logs on your Kubernetes cluster
+`here <https://www.deepnetwork.com/blog/2020/01/27/ELK-stack-filebeat-k8s-deployment.html>`_.
+
+As of 1.18.0, the :ref:`HTTP JSON API <json-api>` component in the Helm chart produces
+JSON-encoded logs. Other components log in somewhat less structured formats.
+
+Daml Metrics Options
+--------------------
+
+The Daml Driver for PostgreSQL instance and the HTTP JSON API instances started
+by the Helm chart are configured to expose Prometheus metrics on a port named
+`metrics`, using the appropriate annotations. This means that, if you are
+running a cluster-wide Prometheus instance, the relevant metrics should be
+collected automatically.
+
+See each component's documentation for details on the metrics exposed:
+
+- `Daml Driver for PostgreSQL </daml-driver-for-postgresql/#types-of-metrics>`_
+- :ref:`JSON API metrics <json-api-metrics>`
+
+Upgrading
+---------
+
+.. note::
+
+   This section only makes sense with the ``production`` flag set to ``true``.
+
+Upgrading the Daml Connect version should be done by uninstalling the exiting
+Helm chart, waiting for all of the pods to stop, and then installing a higher
+version. Destroying all of the components is a safe operation because all of
+the state is stored in the provided database coordinates. There is no
+additional state within the components themselves.
+
+The components are not designed for running concurrently with older versions,
+so it is imperative to wait for the existing Helm chart components to be
+completely shut down before installing the new one. Do not try to upgrade in
+place.
+
+Assuming you do not change the database coordinates, you should have data
```

Comment:
only concerned about the json api here but in theory it should be fine

Reply:
Would you mind expanding on that concern?

Reply:
after reading it again I think it is clear enough